doc/content/integrations/storage/enable.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Enable
+description: ""
+summary: Enable Storage Integration for applications and end-devices.
+weight: 20
+---
+
+The Storage Integration is implemented as an [Application Package]({{< ref "/integrations/application-packages" >}}). In order to enable it for an application (or a single end device), you have to create a package association.
+
+{{< cli-only >}}
+
+## Enable for an Application
+
+Set up a default association between the desired application and the `storage-integration` package.
+
+```bash
+$ ttn-lw-cli applications packages default-associations set "app1" 100 --package-name storage-integration
+```
+
+> *NOTE*: The `f_port` for the default association is set to `100`. This value is irrelevant. **The storage integration will receive and store all uplink messages, regardless of `f_port`**. The `f_port` value is only needed because of the way application packages work with {{% tts %}}.
```

Comment:
```suggestion
> **Note**: The `f_port` for the default association is set to `100`. This value is irrelevant. **The storage integration will receive and store all uplink messages, regardless of `f_port`**. The `f_port` value is only needed because of the way application packages work with {{% tts %}}.
```